### ISSUE TITLE
A11y Accessibility: After applying reflow, "Editor content" textbox is not visible

### ISSUE THREAD

**Issue by joe-braley** [konveyor/static-report#133]
Steps to Reproduce

1.  Install Kantra tool and execute an assessment.
2.  With the report open:
     - TAB until "hamburger" icon and press ENTER key.
     - TAB until "Issues" tab and press ENTER key.
     - TAB until "Detail" key (`>`) in first column and press ENTER key.
     - TAB until first file entry (e.g., pom.xml) and press ENTER key.
     - TAB until "Editor content" text box.
     - Apply Reflow:
         - Open Chrome/Edge dev tools.
         - Undock dev tools so that it’s a separate window.
         - Zoom the browser to 200%
         - Adjust browser width and height to 320 x 256. As long as dev tools remain open while resizing the viewport size will be displayed in the upper right corner of the browser so the size can be verified.
      - Observe that after applying reflow, after applying resize, "Editor content" textbox is not visible

Expected Behavior
After applying resize, "Editor content" textbox should be visible.

Actual Behavior
After applying resize, "Editor content" textbox is not visible.

Environment Details
	• Operating System: Windows 11 Enterprise
        - Software Version: Microsoft Edge Version 130.0.2849.68 (Official build) (64-bit)

Other Resources
![Image](https://github.com/user-attachments/assets/564955c3-586f-443c-9321-0d6af82b9607)

**Comment by joe-braley** [comment]
CC @kthatipally
